macos/Localization/be.xliff

```diff
@@ -314,27 +314,27 @@
       </trans-unit>
       <trans-unit id="Presentation template" xml:space="preserve">
         <source>Presentation template</source>
-        <target state="translated">Шаблон прэзентацыі PowerPoint Open XML (*.potx)</target>
+        <target state="translated">Шаблон прэзентацыі</target>
```

Comment:
correct source for translation: PowerPoint Open XML Presentation Template (*.potx)